### PR TITLE
Update Owl README.md

### PR DESCRIPTION
**Pull request by yasmineevjen**
Update Owl README.md

### CODE REVIEW

File: Owl/README.md [hunk 1]
```diff
@@ -3,19 +3,18 @@
...
+This project is an Android implementation of [Owl](https://material.io/design/material-studies/owl.html), a Material Study showcasing the possibilities of using Material Theming and Material Components for Android.
 
 ## Screenshots
 <img src="screenshots/owl_demo.gif" height="400" alt="Owl sample demo"/>
 
 ## Material Theming
-Owl employs material theming to reflect the energy and excitement of learning a new skill, using a bold aesthetic and unfilled shapes that invite the user to populate them with new content and courses. The app customizes [color](https://material.io/develop/android/theming/color/), [shape](https://material.io/develop/android/theming/shape/) and [typography](https://material.io/develop/android/theming/typography/) to achieve this.
+Owl uses Material Theming to customize the app’s [color](https://material.io/develop/android/theming/color/), [shape](https://material.io/develop/android/theming/shape/) and [typography](https://material.io/develop/android/theming/typography/) to achieve this.
```
Comment (hunterstich) [inline Owl/README.md]: Remove `to achieve this` at the end of this sentence?

File: Owl/README.md [hunk 2]
```diff
@@ -3,19 +3,18 @@
...
 
-Owl has three primary colors. Each color is used to create a distinct visual theme for each section. See [colors.xml](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/color.xml) and how they are used in [light](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/theme.xml#L58-L86) and [dark](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values-night/theme.xml) themes.
+Owl has three primary colors. Each color is used to create a distinct visual theme for each section. See [colors.xml](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/color.xml) and how they are used in [default](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/theme.xml#L58-L86) and [dark](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values-night/theme.xml) themes.
```
Comment (hunterstich) [inline Owl/README.md]: `[colors.xml]` should be `[color.xml]`

Not from your changes, but might as well fix it while we're updating!

Comment (hunterstich) [reply Owl/README.md]: What do you think about adding a bit more to `and how they are used..`?

Like: `See color.xml for the full palette and how it is applied in default and dark themes.`?

File: Owl/README.md (hunk 1 above)
Comment (yasmineevjen) [reply Owl/README.md]: Good catch! Agreed, remove "to achieve this"

File: Owl/README.md (hunk 2 above)
Comment (yasmineevjen) [reply Owl/README.md]: Updated this paragraph to read "Owl has three primary colors which are used to create distinct visual themes for each section. See color.xml for the full color scheme and how colors are applied across default and dark themes."

**Comment by nickbutcher** [comment]
Thanks @yasmineevjen—hope you don't mind me merging but I wanted to build on top of this change.

**Pull request merged**